server/src/utils/jobScraper.js

```diff
@@ -0,0 +1,57 @@
+import FeedParser from 'feedparser';
+import request from 'request';
+
+let getJobsFromStackOverflow = () => {
+  // make a request to Stack Overflow for jobs data
+  let req = request('https://stackoverflow.com/jobs/feed?searchTerm=JavaScript&location=San+Francisco&sort=p');
+  //Instantiate RSS feedparser
+  let parser = new FeedParser();
+
+  req.on('error', function(err) {
+    throw err;
+  })
+
+  req.on('response', function(res) {
+    if (res.statusCode != 200) return this.emit('error', new Error('Bad status code'));
```

Comment:
Also, killing me with this one line return statement without curly braces. Cmon breh